packages/theming/src/index.js

```diff
@@ -6,7 +6,10 @@
  */
 
 export { default as ThemeProvider } from './ThemeProvider/ThemeProvider';
+export { default as defaultTheme } from './theme';
```

Comment:
As the theme and palette are constants, should they be exported as `DEFAULT_THEME` and `PALETTE`